packages/ember-runtime/lib/mixins/mutable_array.js

```diff
@@ -128,9 +145,9 @@ export default Mixin.create(EmberArray, MutableEnumerable, {
     ```javascript
     var colors = ['red', 'green', 'blue', 'yellow', 'orange'];
 
-    colors.removeAt(0);     // ['green', 'blue', 'yellow', 'orange']
-    colors.removeAt(2, 2);  // ['green', 'blue']
-    colors.removeAt(4, 2);  // Error: Index out of range
+    removeAt(colors, 0);     // ['green', 'blue', 'yellow', 'orange']
```

Comment:
These examples are showing how to use the mutable array mixin itself, I would think that we want to leave them as `colors.removeAt(2, 2)`.